rpc/v1/openapi.yaml

```diff
@@ -130,110 +218,1622 @@ paths:
           content:
             text/plain:
               example: "description of failure"
-  /v1/query/allChainParams:
+
+  /v1/query/account:
+    post:
+      tags:
+        - query
+      summary: Returns the data of the account specified at a given height
+      requestBody:
+        description: Request account data at the specified height, height = 0 is used as the latest
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/QueryAddressHeight"
+            example:
+              address: da034209758b78eaea06dd99c07909ab54c99b45
+              height: 0
+        required: true
+      responses:
+        "200":
+          description: Returns account data at the specified height
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Account"
+        "400":
+          description: Bad request
+          content:
+            text/plain:
+              example: "description of failure"
+        "500":
+          description: An error occurred while retrieving the account data at the specified height
+          content:
+            text/plain:
+              example: "description of failure"
+  /v1/query/accounts:
+    post:
+      tags:
+        - query
+      summary: Returns the paginated data of all accounts specified at a given height
+      requestBody:
+        description: Request all account data at the specified height, height = 0 is used as the latest; Max per_page=1000
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/QueryHeightPaginated"
```

Comment:
s/QUeryHeightPaginated/QueryAddressHeightPaginated/

Reply:
The `QueryHeightPaginated` is for things like get _all_ apps and doesnt make sense to include an address here.